addons/circulation/stringtable.xml

```diff
@@ -1460,7 +1460,7 @@
         <Key ID="STR_kat_circulation_TRAINING_LEVEL_AED_X">
             <English>Training level required to use an AED-X</English>
             <Polish>Poziom wyszkolenia wymagany do korzystania z AED-X</Polish>
-            <German>Benötigtes Trainingslevel, um den AED-X zu nutzen</German>
+            <German>Benötigter medizinischer Grad, um den AED-X zu nutzen</German>
```

Comment:
IDK about changing that one, if it should be changed them pls in every other string table as well and not just randomly in this one.

Reply:
I changed them in all, currently it is random...